x/dsmr/block.go

```diff
@@ -119,3 +127,28 @@ type Block struct {
 func (b Block) GetID() ids.ID {
 	return b.blkID
 }
+
+func (b Block) Parent() ids.ID {
+	return b.ParentID
+}
+
+func (b Block) Timestamp() int64 {
+	return b.Tmstmp
+}
+
+func (b Block) Height() uint64 {
+	return uint64(b.Hght)
+}
+
+func (b Block) Txs() []*ChunkCertificate {
+	return b.ChunkCerts
+}
+
+func (b Block) ContainsTx(id ids.ID) bool {
```

Comment:
Can we change this name since chunks are not transactions?

Reply:
done